include/pisa/topk_queue.hpp

```diff
@@ -10,7 +10,7 @@ using Threshold = float;
 struct topk_queue {
     using entry_type = std::pair<float, uint64_t>;
 
-    explicit topk_queue(uint64_t k) : m_threshold(0), m_k(k) { m_q.reserve(m_k + 1); }
+    explicit topk_queue(uint64_t k) : m_threshold(0), m_predicted_threshold(0), m_k(k) { m_q.reserve(m_k + 1); }
```

Comment:
Minor: 
`topk_queue(uint64_t k, Threshold t)` can we have a constructor where we pass an initial threshold so we dont need to set it later?

Reply:
Aggreed.